indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -95,5 +176,43 @@
                 });
             }
         });
+
+        $('.email-preview-btn').on('click', function(evt) {
+            evt.preventDefault();
+            var id = $(this).data('id');
+            var $previewBtn = $('#email-preview-btn-' + id);
+            if ($previewBtn.data('visible')) {
+                $previewBtn.text($T.gettext('Show preview'));
+                $('#email-preview-' + id).slideToggle();
+                $previewBtn.data('visible', false);
+            } else {
+                $previewBtn.text($T.gettext('Hide preview'));
+                $('#email-preview-' + id ).slideToggle();
+                $previewBtn.data('visible', true);
+            }
+        });
+
+        $('#email-template-manager .ui-i-box-sortable-handle').on('mousedown', function() {
+            $('.email-preview').hide();
+            $('.email-preview-btn').text($T.gettext('Show preview')).data('visible', false);
+        })
+
+        $('.js-toggle-stop-on-match').on('click', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+            var stopOnMatch = !$this.data('stop-on-match');
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                data: JSON.stringify({stop_on_match: stopOnMatch}),
+                dataType: 'json',
+                contentType: 'application/json',
+                error: handleAjaxError,
```

Comment:
should we also show the progress dialog while the request is running?

If yes, add:

    complete: IndicoUI.Dialogs.Util.progress(),